apps/files/src/components/Collaborators/Collaborator.vue

```diff
@@ -3,9 +3,33 @@
     <oc-table-row v-if="$_reshareInformation" class="files-collaborators-collaborator-table-row-top">
       <oc-table-cell shrink :colspan="firstColumn ? 2 : 1"></oc-table-cell>
       <oc-table-cell colspan="2">
-        <div class="uk-text-meta uk-flex uk-flex-middle">
-          <oc-icon name="repeat" class="uk-preserve-width" />
-          <span class="uk-padding-remove uk-margin-xsmall-left uk-text-truncate files-collaborators-collaborator-reshare-information">{{ $_reshareInformation }}</span>
+        <div class="uk-text-meta">
+          <a :id="$_resharerToggleId" :aria-label="$gettext('Show resharer details')" class="uk-flex uk-flex-middle">
```

Comment:
We have the `raw` variation of the button where the styling is the same as a link, right? Use that instead pls

Reply:
the raw variation still capitalizes the text :-/

Reply:
I see. I'd see that as an issue to be raised in ODS.

Reply:
Ah, we already have one - https://github.com/owncloud/owncloud-design-system/issues/442

Reply:
adjusted to use oc-button

however we'll need to adjust ODS to also not force a height on raw buttons: https://github.com/owncloud/owncloud-design-system/issues/654